website/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.0",
   "author": "HashiCorp",
   "dependencies": {
-    "@hashicorp/mktg-global-styles": "2.1.0",
```

Comment:
Going to upgrade to a stable release in the assembly once we have all the color values updated.